recipes/tensorflow-lite/all/patches/dependencies.patch

```diff
@@ -0,0 +1,124 @@
+diff --git a/tensorflow/lite/CMakeLists.txt b/tensorflow/lite/CMakeLists.txt
+index 0924ca9dacd..fed88560a02 100644
+--- a/tensorflow/lite/CMakeLists.txt
++++ b/tensorflow/lite/CMakeLists.txt
+@@ -146,37 +146,16 @@ set(TFLITE_TARGET_DEPENDENCIES "")
+ find_package(absl REQUIRED)
+ find_package(Eigen3 REQUIRED)
+ find_package(farmhash REQUIRED)
+-find_package(fft2d REQUIRED)
++find_package(fft REQUIRED)
```

Comment:
You can probably do `deps = ConanDeps(self); deps.set_property("fft", "cmake_file_name", "fft2d")` instead.

Reply:
The same goes for xnnpack.